config/log4j2.properties

```diff
@@ -0,0 +1,82 @@
+status = error
```

Comment:
Did you make any changes to this file or is the same that ES ships already?

What we want is less verbose logging and to console only.